scripts/generate-embedded-metadata-test-files.py

```
@@ -175,6 +197,63 @@ def generate_webm():
+    )
+    generate_av_fixture(
+        'with_nan_metadata.mp3', 'mp3', 'libmp3lame',
+        prompt_json=PROMPT_NAN_JSON, workflow_json=None,
+    )
+    generate_av_fixture(
+        'with_nan_metadata.webm', 'webm', 'libvorbis',
+        prompt_json=PROMPT_NAN_JSON, workflow_json=None,
+    )
+
+    path = out('with_nan_metadata.mp4')
+    subprocess.run([
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)